uktrade/data-hub-frontend
# Readmore component...

## Description of change
As we now include export enquiries into the activity feed there is a user need to toggle the size of the "comment" contents as there is no set limit as to what can be inputted via Great.gov forms. I have created a `<Readmore>` component which allows you to truncate the contents at a character limit of 255 (for readability and accessibility it also stops at the last whole word), the content then ends with ellipses "..." and a "Read more" link shows which allows you to toggle the full contents. 

You can also specify a count incase you want to truncate less or more than 255 characters.

## Test instructions
You can see the component in Storybook however if you run the app against sandbox, choose a company and click on "All external activity" you should see one export enquiry.

## Screenshots
![Screenshot 2020-12-14 at 15 11 53](https://user-images.githubusercontent.com/10154302/102099248-00e7d100-3e20-11eb-870b-6274194cb1da.png)
![Screenshot 2020-12-14 at 15 12 00](https://user-images.githubusercontent.com/10154302/102099259-05ac8500-3e20-11eb-8955-30e2310b4915.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
